workingDirectory.txt

```diff
@@ -0,0 +1,4 @@
+README.md
```

Comment:
The goal of this was to product the working directory path.  I will accept this as it is a bit vague instruction but for future reference your current path is `pwd`